### PR TITLE
add alternate IPL path

### PR DESCRIPTION
**Pull request by haolp**

### CODE REVIEW

File: smtLayer/makeVM.py [hunk 1]
```diff
@@ -151,7 +152,8 @@ def createVM(rh):
         dirLines.append("COMMAND ATTACH PCIF %s * AS %s" % (s[0], s[1]))
 
     if 'ipl' in rh.parms:
-        ipl_string = "IPL %s " % rh.parms['ipl']
+        dirLines.append("IPL LOADDEV")
+        ipl_string = "LOADDEV DEVICE %s " % rh.parms['ipl']
```
Comment (dongyanyang) [inline smtLayer/makeVM.py:156]: ipl_string should be "IPL LOADDEV"
The iplparam and iplLoadparam is for the IPL line.

**Review by dongyanyang — CHANGES_REQUESTED** [review]

File: smtLayer/makeVM.py (hunk 1 above)
Comment (haolp) [reply smtLayer/makeVM.py]: updated, thanks.

File: smtLayer/makeVM.py [hunk 2]
```diff
@@ -160,6 +161,7 @@ def createVM(rh):
             ipl_string += ("LOADPARM %s " % rh.parms['iplLoadparam'])
 
         dirLines.append(ipl_string)
+        dirLines.append("LOADDEV DEVICE %s " % rh.parms['ipl'])
```
Comment (bjhuangr) [inline smtLayer/makeVM.py:164]: Is it compatible with DASD root disk with `IPL LOADDEV` + `LOADDEV DEVICE`

And `IPL LOADDEV PARM XXX` `IPL LOADDEV LOADPARM XXX`?

Comment (haolp) [reply smtLayer/makeVM.py]: update to not make change for dasd, thanks for your comment.

**Review by dongyanyang — APPROVED** [review]

**Review by bjhuangr — APPROVED** [review]

**Comment by bjhuangr** [comment]
@haolp you need another PR to change the API doc for this parameter update

**Comment by haolp** [comment]
> @haolp you need another PR to change the API doc for this parameter update

got it, I will submit it later.

**Comment by haolp** [comment]
> @haolp you need another PR to change the API doc for this parameter update

@bjhuangr I realized that this PR did not add new parameter when create guest, just put more data in existing parameter 'loaddev', so seems no need to update the doc?

```
.. restapi_parameters:: parameters.yaml

  - guest: guest_dict
  - userid: userid_create
  - vcpus: guest_vcpus
  - memory: guest_memory
  - user_profile: user_profile_guest
  - disk_list: disk_list_guest
  - size: size_disk
  - format: format_disk
  - is_boot_disk: is_boot_disk
  - vdev: add_mdisk_disk_vdev
  - disk_pool: disk_pool_guest
  - max_cpu: guest_max_cpu
  - max_mem: guest_max_mem
  - ipl_from: guest_ipl_from
  - ipl_param: guest_ipl_param
  - ipl_loadparam: guest_ipl_loadparam
  - dedicate_vdevs: guest_dedicate_vdevs
  - loaddev: guest_loaddev     <------
  - account: guest_account
  - comment_list: guest_comment_list
```

**Pull request merged**